content/attacks/posts/2022-08-01-Nomad.md

```diff
@@ -0,0 +1,31 @@
+---
+date: 2022-08-01
+custodians: Nomad
+categories: Bridge Hack
```

Comment:
Please, reformat headers regarding with [new changes](https://github.com/1712n/dn-institute/tree/main#readme). Here is an example:
```
target-entities: Nomad
entity-types:
- DeFi
- Exchange
attack-types:
- Smart Contract Exploit
```